### PR TITLE
Move bot url to botbuild

### PR DESCRIPTION
**Pull request by netroby**
Move bot url to botbuild 

See https://github.com/Araq/Nim/issues/2293
and https://github.com/Araq/Nim/issues/1765

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -9,16 +9,16 @@ the C source of an older version of the compiler are needed to bootstrap the
 latest version. The C sources are available in a separate repo [here](http://github.com/nim-lang/csources).
 
 Pre-compiled snapshots of the compiler are also available on
-[Nimbuild](http://build.nim-lang.org/). Your platform however may not 
+[Nimbuild](http://buildbot.nim-lang.org/). Your platform however may not
```
Comment (dom96) [inline readme.md]: This should be changed. It's not longer Nimbuild it's a python build-bot.

Comment (dom96) [inline readme.md]: Also, I don't think it provides pre-compiled snapshots. So maybe you should just remove it?

**Comment by netroby** [comment]
Provided us pre-compiled snapshots would be very appreciate.
Do not just remove it.

For example. Blender release snapshots build for users.

https://builder.blender.org/

we can help for testing the latest snapshots. see if there some bugs or what ever.

**Comment by reactormonk** [comment]
Make a new issue about builds.

**Pull request merged**